src/C-interface/ellblock/bml_allocate_ellblock_typed.c

```diff
@@ -183,13 +183,17 @@ bml_matrix_ellblock_t *TYPED_FUNC(
     int mb = bml_get_mb();
     int nb = bml_get_nb();
 
+    printf("N = %d, M = %d, len(bsize) = %d\n", N, M, nb);
```

Comment:
This type of debugging info should not be merged with master!

Reply:
Definitely not @jeanlucf22 . My intention was to show you what debug statements I had inserted, and I should have marked this PR clearly as "do not merge".